pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -126,6 +135,26 @@ func TestValidateKubeSchedulerConfiguration(t *testing.T) {
 	oneEmptyQueueSort := validConfig.DeepCopy()
 	oneEmptyQueueSort.Profiles[0].Plugins = nil
 
+	extenderNegativeWeight := validConfig.DeepCopy()
+	extenderNegativeWeight.Extenders[0].Weight = -1
+
+	extenderDuplicateManagedResource := validConfig.DeepCopy()
+	extenderDuplicateManagedResource.Extenders[0].ManagedResources = []config.ExtenderManagedResource{
+		{Name: "foo", IgnoredByScheduler: false},
+		{Name: "foo", IgnoredByScheduler: false},
+	}
+
+	extenderDuplicateBind := validConfig.DeepCopy()
+	extenderDuplicateBind.Extenders[0].BindVerb = "foo"
+	extenderDuplicateBind.Extenders = append(extenderDuplicateBind.Extenders, config.Extender{
+		URLPrefix:      "http://localhost2:8888/",
+		FilterVerb:     "filter",
+		PrioritizeVerb: "prioritize",
+		BindVerb:       "bar",
+		Weight:         1,
+		EnableHTTPS:    false,
+	})
+
```

Comment:
where are those configurations being used in the tests?

Reply:
Ah I didn't need all those fields, and I forgot to enable these tests... fixed